spec/routing/work_packages_spec.rb

```diff
@@ -44,8 +44,7 @@
     expect(get('/work_packages/1/overview'))
       .to route_to(controller: 'work_packages',
                    action: 'index',
-                   id: '1',
-                   state: 'overview')
+                   state: '1/overview')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.